src/test/java/uk/gov/pay/ledger/transaction/search/common/TransactionSearchParamsValidatorTest.java

```diff
@@ -45,14 +45,14 @@ public void shouldNotThrowException_whenValidDateFormats() {
     @Test
     public void shouldNotThrowException_whenGatewayAccountIdAvailableAndWithParentTransactionIsTrue() {
         searchParams.setWithParentTransaction(true);
-        TransactionSearchParamsValidator.validateSearchParams(searchParams,"account-id");
+        TransactionSearchParamsValidator.validateSearchParams(searchParams, new CommaDelimitedSetParameter("account-id"));
     }
 
     @Test
     public void shouldThrowException_whenGatewayAccountIdIsNotAvailableAndWithParentTransactionIsTrue() {
         searchParams.setWithParentTransaction(true);
         thrown.expect(ValidationException.class);
         thrown.expectMessage("gateway_account_id is mandatory to search with parent transaction");
-        TransactionSearchParamsValidator.validateSearchParams(searchParams,"");
+        TransactionSearchParamsValidator.validateSearchParams(searchParams,new CommaDelimitedSetParameter(""));
```

Comment:
A test with `new CommaDelimitedSetParameter("1,2,3")` would be good.

Reply:
done!!